docs/HELM.md

```diff
@@ -26,10 +27,18 @@ $ helm install --set logdna.key=$LOGDNA_INGESTION_KEY my-release logdna/agent
 
 You should see logs in https://app.logdna.com in a few seconds.
 
+### Using a namespace
+
+If you want to scope your release to a namespace, you can use Helm's `-n` flag when installing:
+
+```bash
+helm install --set logdna.key=$LOGDNA_INGESTION_KEY -n my-namespace --create-namespace my-release logdna/agent
+```
+
 ### Tags support:
 
 Optionally, you can configure the LogDNA Agent to associate tags to all log records that it collects so that you can
-identify the agent's data quicker in the LogDNA web UI.
+identify the data quicker in the LogDNA web UI.
```

Comment:
It would be best to say "more quickly" instead of "quicker."

Reply:
Thanks, I'll change it!